buildSrc/src/main/java/org/opensearch/gradle/PublishPlugin.java

```diff
@@ -146,9 +148,36 @@ public String call() throws Exception {
 
     private static void addScmInfo(XmlProvider xml) {
         Node root = xml.asNode();
-        root.appendNode("url", Util.urlFromOrigin(BuildParams.getGitOrigin()));
-        Node scmNode = root.appendNode("scm");
-        scmNode.appendNode("url", BuildParams.getGitOrigin());
+        Node url = null, scm = null;
+
+        for (final Object child : root.children()) {
```

Comment:
Thanks @reta for picking this up, is there a way to add tests ?

Reply:
I think it is possible, but takes some time ... will look into it